src/styles/month.scss

```diff
@@ -0,0 +1,65 @@
+@import 'variables';
+
+.calendar {
+  width: $cellsize * 7 + 6px;
```

Comment:
The calculation for the width of the calendar might be incorrect if $cellsize is not defined in a way that accounts for the 6px gap. Ensure that $cellsize is defined correctly in 'variables.scss'.